packages/imperative/src/cmd/__tests__/CommandProcessor.unit.test.ts

```diff
@@ -1679,54 +1618,6 @@ describe("Command Processor", () => {
         expect(commandResponse.data.locations.length).toBeGreaterThan(0);
         expect(commandResponse.data.optionalProfiles[0]).toBe(`banana`);
         expect(commandResponse.data.requiredProfiles).toBeUndefined();
-        expect(commandResponse.data.profileVersion).toBe('TeamConfig');
-    });
-
-
-    it("should mask input value for a secure parm when --show-inputs-only flag is set", async () => {
```

Comment:
Is removing this test deliberate? It doesn't look specific to v1 profiles to me, so seems like it would be valid to keep.

Reply:
Even though it doesn't look like it is looking for V1 profiles, the fact that it was not defining a team_config meant that it was always falling back to V1 profiles. (hint: `commandParameters.definition.profile` isn't populated by the `prepare` method)

Happy to discuss offline 😋

Reply:
```it("should display input value for simple parm when --show-inputs-only flag is set and v1 profile exists", async () => {"```
yes, this test is related to setting --show-inputs-only when using v1 and my understanding is that this PR is about migrating away from v1 cases meaning this situation will no longer be present in v3

Reply:
I don't see any test with that name in [this file](https://github.com/zowe/zowe-cli/blob/show-inputs-only/packages/imperative/src/cmd/__tests__/CommandProcessor.unit.test.ts)

Reply:
and team config exists****

Reply:
` it("should mask input value for a secure parm when --show-inputs-only flag is set", async () => {` should resolve this conversation. sorry for confusion